libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -114,14 +114,24 @@ public UserIdentityService(ConfigService configService, DataStore<VcStoreItem> d
         VcHelper.setConfigService(configService);
     }
 
+    public List<VcStoreItem> getVcStoreItems(String userId) {
+        return dataStore.getItems(userId);
+    }
+
+    public VcStoreItem getVcStoreItem(String userId, String criId) {
+        return dataStore.getItem(userId, criId);
+    }
+
     public List<String> getUserIssuedCredentials(String userId) {
```

Comment:
Similar to below, I wonder if it would be better to remove this signature - it will mean that the two existing usages have to add an extra line to fetch the credentials separately, but it makes it easy to spot if you start fetching multiple times.

What do you think?